src/include/firewall/uifunctions.rb

```diff
@@ -135,21 +144,33 @@ def SaveAndRestart
     def StartNow
       UI.OpenDialog(Label(_("Starting firewall...")))
       SuSEFirewall.SetStartService(true)
-      ret = SuSEFirewall.StartServices
+      success = SuSEFirewall.StartServices
+      Popup.Error(
+        _(
+          "The firewall could not be started.\n" +
+          "Please verify your system and try again."
+        )
+      ) unless success
       UI.CloseDialog
 
-      ret
+      success
     end
 
     # Function stops Firewall services and sets firewall
     # to be stopped after exiting YaST
     def StopNow
       UI.OpenDialog(Label(_("Stopping firewall...")))
       SuSEFirewall.SetStartService(false)
-      ret = SuSEFirewall.StopServices
+      success = SuSEFirewall.StopServices
+      Popup.Error(
+        _(
+          "The firewall could not be stopped.\n" +
+          "Please verify your system and try again."
+        )
```

Comment:
On the other hand the original version is more generic, imagine a language where words "start" and "stop" use a different grammar rules and a simple `%s` placeholder might not be enough.

The question is whether we want to be rather more generic or make the work for translators easier...